contents/docs/integrate/vercel-marketplace.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Vercel Marketplace Integration
+sidebarTitle: Vercel Marketplace
+icon: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/vercel_icon_svgrepo_com_b7e78b41f9.svg
+---
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+The Vercel Marketplace integration adds PostHog feature flags and experiments to your Vercel projects with billing handled through Vercel.
+
+Choose this integration if you do not already have a PostHog account or you prefer to consolidate payment for PostHog inside your Vercel invoice.
+
+## Installing the integration
+
+1. Go to the [PostHog integration](https://vercel.com/integrations/posthog) on Vercel Marketplace
+2. Click **Install**
+
+This creates a PostHog organization and project, and adds environment variables to your Vercel project.
+
+## Environment variables
+
+The integration adds:
+- `POSTHOG_PROJECT_API_KEY` - Your PostHog project API key
+- `POSTHOG_HOST` - PostHog API host URL
+
+These names work across all frameworks. For client-side frameworks, rename them in your Vercel project settings:
```

Comment:
This is a bit weird. Can we make these work automatically? Vercel should set them up with the `NEXT_PUBLIC` prefix by default